src/GraphQL.Tests/Serialization/ExecutionResultTests.cs

```diff
@@ -135,21 +135,17 @@ public void Writes_Correct_Execution_Result_With_Empty_Data_Errors_And_Extension
 
         [Theory]
         [ClassData(typeof(GraphQLSerializersTestData))]
-        public void Writes_Path_Property_Correctly(IGraphQLTextSerializer serializer)
+        public void Writes_Correct_Execution_Result_With_Null_Data_Errors_And_Extensions_When_Executed(IGraphQLTextSerializer serializer)
```

Comment:
`Writes_Path_Property_Correctly` was moved to `ExecutionErrorTests.cs`.  This is a new test that verifies that `data` is serialized when `Executed` is `true` even if it is `null`.